utils/_context/_scenarios/debugger.py

```diff
@@ -0,0 +1,32 @@
+import pytest
+
+from .core import ScenarioGroup
+from .endtoend import EndToEndScenario
+
+
+class DebuggerScenario(EndToEndScenario):
+    def __init__(self, name: str, doc: str, weblog_env: dict[str, str | None]) -> None:
+        base_weblog_env: dict[str, str | None] = {
+            "DD_REMOTE_CONFIG_ENABLED": "1",
+            "DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS": "1",
+        }
+
+        base_weblog_env.update(weblog_env)
+
+        super().__init__(
+            name=name,
+            doc=doc,
+            rc_api_enabled=True,
+            library_interface_timeout=5,
+            weblog_env=base_weblog_env,
+            scenario_groups=[ScenarioGroup.DEBUGGER],
+        )
+
+    def configure(self, config: pytest.Config):
+        super().configure(config)
+
+        library = self.weblog_container.image.labels["system-tests-library"]
+        if library == "python":
+            self.weblog_container.environment["DD_DYNAMIC_INSTRUMENTATION_UPLOAD_FLUSH_INTERVAL"] = "0.1"
+        else:
+            self.weblog_container.environment["DD_DYNAMIC_INSTRUMENTATION_UPLOAD_FLUSH_INTERVAL"] = "100"
```

Comment:
The Node.js tracer at least doesn't support this environment variable. Should we make sure all tracers support it?

Reply:
This is a question we should discuss on agent sync. I put a topic for this.